app/views/menu/index.html.erb

```diff
@@ -1,12 +1,14 @@
 <h1>Menu</h1>
 <div class="row">
   <% @items.each do |item| %>
-    <div class="col-sm-4 items">
+    <div class="col-sm-3 items">
       <%= item.name %>
-      <%= button_to "Add to Cart", cart_items_path(item: item) %>
+      <%= button_to "Add to Cart", cart_items_path(item: item), class: "btn btn-default" %>
+      <div class="btn-group btn-group-xs" role="group">
       <% item.categories.each do |item_category| %>
-      <p><%= link_to item_category.display_name, menu_category_path(item_category) %></p>
+        <%= link_to item_category.display_name, menu_category_path(item_category), class: "btn btn-default" %>
```

Comment:
Can you refer to these as just `item_category` instead of `menu_category_path(item_category)`?